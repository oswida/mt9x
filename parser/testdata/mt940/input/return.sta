:20:151006120035300006
:25:987654321
:28C:112/1
:60F:C151006SEK89324,89
:61:1510061006D300,NSWRORIGINALAVSANDAR//BGC1234567890002
:86:/REMI/Meddelande som kan vara max 50 tecken/ORDP/1234567899
/BENM/+46709876543 ORIGINALAVSANDARENS NAMN 
:62F:C151006SEK89024,89
:64:C151006SEK89024,89