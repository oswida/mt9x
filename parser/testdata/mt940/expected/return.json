{
 "tag20": "151006120035300006",
 "tag25": {
  "account": "987654321"
 },
 "tag28": {
  "stmt_number": "112",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2015-10-06T00:00:00Z",
  "currency": "SEK",
  "amount": "89324.89"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2015-10-06T00:00:00Z",
    "entry_date": "0000-10-06T00:00:00Z",
    "dc_mark": "D",
    "amount": "300",
    "trx_ident": "NSWR",
    "owner_ref": "ORIGINALAVSANDAR",
    "institution_ref": "BGC1234567890002"
   },
   "tag86": [
    "/REMI/Meddelande som kan vara max 50 tecken/ORDP/1234567899",
    "/BENM/+46709876543 ORIGINALAVSANDARENS NAMN "
   ]
  }
 ],
 "tag62": {
  "dc_mark": "C",
  "date": "2015-10-06T00:00:00Z",
  "currency": "SEK",
  "amount": "89024.89"
 },
 "tag64": {
  "dc_mark": "C",
  "date": "2015-10-06T00:00:00Z",
  "currency": "SEK",
  "amount": "89024.89"
 }
}